tests/test_modeling_tf_led.py

```diff
@@ -362,6 +363,128 @@ def check_encoder_attentions_output(outputs):
             self.assertEqual(model.config.output_hidden_states, True)
             check_encoder_attentions_output(outputs)
 
+    # TODO: Remove this once a more thorough pt/tf equivalence could be implemented in `test_modeling_tf_common.py`.
+    # (Currently, such a test will fail some other model tests: it requires some time to fix them.)
+    @is_pt_tf_cross_test
+    def test_pt_tf_model_equivalence_extra(self):
+        import torch
```

Comment:
@patrickvonplaten  I added this test for TFLED. Is this OK?

Reply:
Perfect!